.changeset/long-poets-wonder.md

```diff
@@ -0,0 +1,6 @@
+---
+"@hashicorp/design-system-components": minor
+---
+
+`Hds::CodeEditor` - Added new CodeMirror 6 supported code editor component
+`hds-code-editor` modifier - Added new code editor modifier which converts the element it is applied to into a CodeMirror6 code editor
```

Comment:
Super nit: Should be consistent as to whether there is a space before the "6" or not in "CodeMirror6".